tasks/collect_info_ent.yml

```diff
@@ -0,0 +1,55 @@
+---
+- block:
+  - name: Get registration token (RUN ONCE)
+    uri:
+      url: "{{ github_api_url }}/enterprises/{{ github_owner | default(github_account) }}/actions/runners/registration-token"
```

Comment:
Every time I see `github_owner`, it is in the context of `github_owner | default(github_account)`

Given that would it be better to set in `defaults/main.yml`
```
github_owner: "{{ github_account }}"
```
?

Reply:
Actually I'm going withdraw this objection.  I can see we are only being consistent with the existing code base.

Reply:
Might be a refactor we can offer at a later date.

Reply:
That, and I think we can remove the assertion that the github_account is defined if the github_owner is already defined.
(Not forgetting your suggestion of splitting tasks into blocks more clearly)